src/lib/util/options.ml

```diff
@@ -259,7 +259,7 @@ let output_with_colors = ref false
 let output_with_headers = ref true
 let output_with_formatting = ref true
 let output_with_forced_flush = ref true
-let frontend = ref "legacy"
+let frontend = ref "dolmen"
 let input_format = ref Native
```

Comment:
Should the default input format be Smtlib2 as well?

Reply:
Somewhat confusingly, the default input format is never actually used, so it doesn't matter. It's fairly easy to clean that up, I will do it in another PR.

Reply:
#858